src/catkin_pkg/package_templates.py

```diff
@@ -294,10 +294,12 @@ def _create_targetlib_args(package_template):
 
 
 def _create_include_macro(package_template):
-    result = '# include_directories(include)'
+    result = '#  include'
```

Comment:
Yes, I wasn't sure how to accomplish that. I thought of just initializing `result` to an empty string and having everything be appended from `includes`?

Regarding the spaces, I was just following what looked like the convention. Here's pasted directly from the first instance of this sort in [CMakeLists.txt.in](https://raw.githubusercontent.com/ros-infrastructure/catkin_pkg/master/src/catkin_pkg/templates/CMakeLists.txt.in):
```
# add_message_files(
#   FILES
#   Message1.msg
#   Message2.msg
# )
```
Let me know what's considered proper style and I'll make it so!